nlp_primitives/lsa.py

```diff
@@ -69,6 +60,13 @@ def lsa(array):
             array = pd.Series(array, index=pd.Series(array.index), name='array')
             copy = array.dropna()
             copy = copy.apply(lambda x: dtk.detokenize(clean_tokens(x)))
+
+            fit_data = copy.tolist()
+            # TruncatedSVD cannot produce two features without multiple input values
+            if len(fit_data) == 1:
+                fit_data = fit_data * 2
```

Comment:
So this case is when only one row has been passed in as input, right? If so, perhaps we should just raise an exception. And then featuretools and evalml can both handle that exception.